sql/dolphinscheduler_mysql.sql

```diff
@@ -248,7 +248,7 @@ CREATE TABLE `QRTZ_TRIGGERS` (
   KEY `IDX_QRTZ_T_NFT_ST_MISFIRE` (`SCHED_NAME`,`MISFIRE_INSTR`,`NEXT_FIRE_TIME`,`TRIGGER_STATE`),
   KEY `IDX_QRTZ_T_NFT_ST_MISFIRE_GRP` (`SCHED_NAME`,`MISFIRE_INSTR`,`NEXT_FIRE_TIME`,`TRIGGER_GROUP`,`TRIGGER_STATE`),
   CONSTRAINT `QRTZ_TRIGGERS_ibfk_1` FOREIGN KEY (`SCHED_NAME`, `JOB_NAME`, `JOB_GROUP`) REFERENCES `QRTZ_JOB_DETAILS` (`SCHED_NAME`, `JOB_NAME`, `JOB_GROUP`)
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
```

Comment:
Why need to use `utf8` to replace `utd8mb4`?

Reply:
https://www.cnblogs.com/cuiqq/p/11045487.html

I read this post and didn't quite understand the need to change the coding format.

Reply:
I checked the qrtz related tables and will not write 4-byte Unicode. Utf8mb4 encoding will cause the index of qrtz related tables to exceed the default value of MySQL, which will cause many users to fail to create tables. 
![image](https://user-images.githubusercontent.com/42576980/92547945-74f66400-f288-11ea-8f24-6161adfb1683.png)

Reply:
> I checked the qrtz related tables and will not write 4-byte Unicode. Utf8mb4 encoding will cause the index of qrtz related tables to exceed the default value of MySQL, which will cause many users to fail to create tables. 
> 
> ![image](https://user-images.githubusercontent.com/42576980/92547945-74f66400-f288-11ea-8f24-6161adfb1683.png)
> 
> 

Which version of your mysql?

Reply:
Version 5.6, I think there are many users of this version of MySQL, we should be compatible

Reply:
Hi,we should discuss first,the utf8mb4 is suggested to use in mysql.You can see the article below.
https://medium.com/@adamhooper/in-mysql-never-use-utf8-use-utf8mb4-11761243e434
I saw the error you pointed `Index column size too large. The maximum column size is 767 bytes.`The another article give a method to resolve it.
https://www.jianshu.com/p/5c0cce24189b

Reply:
I will send a email in the dev list,we can discuss it which charset we need to choose.